package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "sphinx_book_theme",
+  "repository": "https://github.com/executablebooks/sphinx-book-theme",
+  "scripts": {
+    "build": "webpack"
+  },
+  "devDependencies": {
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.1.1",
+    "css-loader": "^3.4.2",
+    "dedent": "^0.7.0",
+    "extract-loader": "^4.0.3",
+    "file-loader": "^5.0.2",
+    "html-webpack-plugin": "^4.3.0",
+    "imports-loader": "^0.8.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "node-sass": "^6.0.1",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
+    "pa11y-ci": "^2.4.0",
```

Comment:
Some of these we can likely clean out (e.g. we do not test with `pa11y-ci`). Let's shoot for the minimal possible dependencies

Reply:
yes. need to do a thorough clean out.